XIV on Mac/de.lproj/Main.strings

```diff
@@ -480,82 +480,82 @@
 "c2g-j3-NbA.title" = "Microsoft .NET 6.0";
 
 /* Class = "NSMenu"; title = "Transformations"; ObjectID = "c8a-y6-VQd"; */
-"c8a-y6-VQd.title" = "Transformations";
+"c8a-y6-VQd.title" = "Verwandlungen";
```

Comment:
Works, but could also be "Transformationen", which is the default system text on German macOS